utils/common-utils.js

```diff
@@ -0,0 +1,95 @@
+import { getItemFromLocalStorage } from "./local-storage-utils.js";
+import { LOCAL_STORAGE_KEYS } from "../constants/common-constants.js";
+import { headerComponent } from "../components/header/header.js";
+
+/**
+ * Retrieves the value of a URL parameter by name.
+ * @param name - The name of the parameter to retrieve.
+ * @returns The value of the parameter if found, or an empty string if not found.
+ */
+export const getParameterByName = (name) => {
+    name = name.replace(/[[]/, "\\[").replace(/[\]]/, "\\]");
+    var regex = new RegExp("[\\?&]" + name + "=([^&#]*)"),
```

Comment:
could you let me know why var is used

Reply:
I have changed that to let, since it is a local scope variable